gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp

```diff
@@ -414,7 +414,7 @@ Renderer11DeviceCaps::Renderer11DeviceCaps() = default;
 
 Renderer11::Renderer11(egl::Display *display)
     : RendererD3D(display),
-      mCreateDebugDevice(false),
+      mCreateDebugDevice(true),
```

Comment:
Is this intentional? We don't want to pay for the debug overhead most of the time.

Reply:
Bah, I thought I removed that change.

Reply:
Fixed.